e2e/constants/selectors/loginselectors.ts

```diff
@@ -0,0 +1,6 @@
+export const LOGINCONSTANTS={
```

Comment:
Two things:
1. Here the object is not just signifying a bunch of random constants. It's signifying a set of selectors. So the constant should be named `Login selectors` and not `Login constants`
2. If we're dealing with a constant name that comprises multiple words, then we should prefer SCREAMING_SNAKE_CASE. Meaning uppercase words separated by underscores(`_`).

```suggestion
export const LOGIN_SELECTORS={
```